python-sdk/src/astro/databases/snowflake.py

```diff
@@ -460,6 +468,21 @@ def is_native_autodetect_schema_available(  # skipcq: PYL-R0201
         )
         return is_file_type_supported and is_file_location_supported
 
+    def create_table(self, table: BaseTable, *args, **kwargs):
+        """Override create_table to add metadata columns to the table if specified in load_options"""
```

Comment:
We should add documentation for loading metadata from Snowflake.

Reply:
We have added it https://github.com/astronomer/astro-sdk/pull/2023/files#diff-6f3049267c42087321bf6e68905f412cc9ccac24a2efdbd0fda0e4967e625d76R74